src/renderer/components/_features/[workspace]/editor/resource-editor/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { GlobalVariablesEditor } from '@root/renderer/components/_organisms/global-variables-editor'
+import InstancesEditor from '@root/renderer/components/_organisms/instances-editor'
+import TaskEditor from '@root/renderer/components/_organisms/task-editor'
+
+export default function ResourcesEditor() {
```

Comment:
Avoid as much as possible using default export notation, as it can lead the code to have unexpected behavior.